### PR TITLE
Improve the manageability of the installer compilation

### PR DESCRIPTION
**Pull request by RestorerZ**
Provides a single installer build management interface from the command line of
the program "makensis.exe".
Allows you to avoid unintended errors during direct manual editing or other ways
of changing the file "gvim.nsi".

### CODE REVIEW

File: nsis/README.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
-This builds a one-click install for Vim for Win32 using the Nullsoft
+This builds a one-click install for Vim for OS Windows using the Nullsoft
```
Comment (k-takata) [inline nsis/README.txt]: I've never seen the wording "OS Windows" in other part of Vim document.
"MS-Windows" might be better?

File: nsis/README.txt [hunk 2]
```diff
@@ -16,10 +16,10 @@ To build the installable .exe:
 	install.exe,
 	uninstall.exe,
 	tee/tee.exe,
-	xxd/xxd.exe,
+	xxd/xxd.exe
 
-    Then execute tools/rename.bat to rename the executables. (mv command is
-    required.)
+    Then execute tools/rename.bat to rename the executables (on Unix-like,
+    it must first be made executable; chmod +x path/to/vimsrc/tools/rename.bat).
```
Comment (k-takata) [inline nsis/README.txt]: How about setting the executable flag in this repository beforehand?

File: nsis/README.txt [hunk 3]
```diff
@@ -61,25 +71,39 @@ To build the installable .exe:
...
+Download Unicode version the ShellExecAsUser plug-in for NSIS from:
+	https://nsis.sourceforge.io/ShellExecAsUser_plug-in
+and put ShellExecAsUser.dll to path\to\NSIS\Plugins\x86-unicode
 
 
 Unpack the images:
 	cd nsis
-	unzip icons.zip
+	unzip icons.zip (on Unix-like)
+	7z.exe x icons.zip or WinRar.exe x icons.zip (on OS Windows)
```
Comment (k-takata) [inline nsis/README.txt]: There is also Windows version of unzip, and there is also Linux version of 7z (p7zip).

Comment (RestorerZ) [reply nsis/README.txt]: I agree. But if in Unix-like systems "unzip" is practically part of the system, then in Windows it is usually only if Gnuwin32 or something similar is installed.
I proceeded from the fact that it is most common in Windows, and 7z is more likely to be installed by the user.
But you're right, it's better to clarify this in the documentation.

File: nsis/README.txt (hunk 2 above)
Comment (RestorerZ) [reply nsis/README.txt]: Oh! That would be great! Could you tell me how to do it? It's just that my main tool is Windows.
I'm going to try to log in to the Ubuntu via VM and set this flag in my fork. Will that be enough?

File: nsis/README.txt (hunk 1 above)
Comment (RestorerZ) [reply nsis/README.txt]: Yes, it would be better that way. MS Windows.

File: nsis/README.txt (hunk 2 above)
Comment (k-takata) [reply nsis/README.txt]: Changing the flag on a unix-like system should be the easiest way and that should be enough.
However, on Windows, you can use `git update-index --add --chmod=+x <filename>`.

Comment (RestorerZ) [reply nsis/README.txt]: Thank you very much! Everything worked out! Thanks!

**Comment by chrisbra** [comment]
Thanks!

**Pull request merged**